pycqed/measurement/pulse_sequences/fluxing_sequences.py

```diff
@@ -360,17 +375,83 @@ def fluxpulse_scope_sequence(
     flux_pulse['name'] = 'FPS_Flux'
     flux_pulse['ref_pulse'] = 'FPS_Pi'
     flux_pulse['ref_point'] = 'middle'
-    flux_pulse['pulse_delay'] = -flux_pulse.get('buffer_length_start', 0)
+    flux_pulse_delays = -delays - flux_pulse.get('buffer_length_start', 0)
```

Comment:
If the aim is to get rid of the buffer, why not just set the buffer length to zero in the deepcopied pulse?

Reply:
This was the case before (in the master branch) with the line 
`flux_pulse['pulse_delay'] = -flux_pulse.get('buffer_length_start', 0)`

This line, however, has no effect, as the pulse lengths are overwritten by
`swept_pulses = sweep_pulse_params(pulses,
                                      {'FPS_Flux.pulse_delay': -delays})`

One therefore has to first define the new delays `flux_pulse_delays` (reduced by the buffer start length) and then use them to sweep the pulse parameters with 
`swept_pulses = sweep_pulse_params(
        pulses, {'FPS_Flux.pulse_delay': flux_pulse_delays})`

Reply:
I meant:
flux_pulse['buffer_length_start'] = 0

Reply:
If you set the buffer to zero, then the gaussian filtered rising edge will be cut off.

Reply:
Yes, I think the way it is solved here is ok in my opinion.